spec/datadog/tracing/contrib/opensearch/patcher_spec.rb

```diff
@@ -0,0 +1,249 @@
+require 'datadog/tracing/contrib/support/spec_helper'
+require 'opensearch'
+
+require 'ddtrace'
+require 'datadog/tracing/contrib/opensearch/integration'
+
+RSpec.describe 'OpenSearch instrumentation' do
+  before do
+    Datadog.configure do |c|
+      c.tracing.instrument :opensearch
+    end
+
+    # Create a client index before every test case
+    unless client.indices.exists?(index: index_name)
+      client.indices.create(
+        index: index_name,
+        body: index_body
+      )
+    end
+
+    clear_traces!
```

Comment:
Interesting, not sure I've seen the need to do this before. Why do we need this?